src/Form/FastlySettingsForm.php

```diff
@@ -244,6 +251,15 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#open' => TRUE,
     ];
 
+    $key_length = (int) $config->get('cache_tag_hash_length') ?: CacheTagsHashInterface::CACHE_TAG_HASH_LENGTH;
```

Comment:
`cache_tag_hash_length` needs to be declared in `fastly.schema.yml`, as well as `cache_tag_prefix`, which should also be configurable in this form.